data/core/macros/abilities.cfg

```diff
@@ -388,6 +388,21 @@ Enemy units cannot see this unit while it is in deep water, except if they have
     [/dummy]
 #enddef
 
+#define ABILITY_EXPLODE TYPE VALUE
+    # Canned definition of the Explode ability to be included in an
+    # [abilities] clause.
+    # This works because of data/lua/explode_ability.lua
+    [dummy]
+        id=explode
+        name= _ "explode"
+        female_name= _ "female^explode"
+        description=_ "This unit explodes upon death, causing damage to any adjacent units."
+        value={VALUE}
+        apply_to={TYPE}
+        special_note={INTERNAL:SPECIAL_NOTES_EXPLODE}
```

Comment:
As I recall, the `INTERNAL:SPECIAL_NOTES_*` macros only exist for backwards compatibility so that the strings can be used in two different places, so I think it would be fine to include the string directly right here.

Reply:
Or keep the macros and use it in both description and special_note, as they seem to mostly say the same thing.

Reply:
I will note that the special not is optional, so you could omit it if you think it isn't something that needs to be mentioned in the unit description (although, exploding _does_ seem notable to me…).

If it does make sense to use the exact same string for description and special note, then yeah, keeping the macro makes sense, but I'm not sure the current name of the macro suits that use…